test/functional/services/remote/webdriver.ts

```diff
@@ -288,14 +289,23 @@ async function attemptToCreateCommand(
   const { session, consoleLog$ } = await buildDriverInstance();
 
   if (throttleOption === '1' && browserType === 'chrome') {
+    const profile = process.env.KBN_NETWORK_TEST_PROFILE ?? 'DEFAULT';
+    const {
+      DOWNLOAD: downloadThroughput,
+      UPLOAD: uploadThroughput,
+      LATENCY: latency,
+    } = NETWORK_PROFILES[`${profile}`];
+
     // Only chrome supports this option.
-    log.debug('NETWORK THROTTLED: 768k down, 256k up, 100ms latency.');
+    log.debug(
+      `NETWORK THROTTLED with profile ${profile}: ${downloadThroughput}kbps down, ${uploadThroughput}kbps up, ${latency} ms latency.`
+    );
 
     (session as any).setNetworkConditions({
```

Comment:
optional nit: it's not related to the current PR but let's use `@ts-expect-error` to detect a moment when selenium webdriver typings add `setNetworkConditions` declaration.